influxdb.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "json"
+  spec.add_runtime_dependency "json", "~> 1.8.3"
```

Comment:
I have added a note for Ruby 1.9 users into the Readme and pinned 1.8.x in the Gemfile in 8a078b3.

Ruby 2.0+ users should not be forced to use an old version of the JSON gem, just because we still want to support Ruby 1.9.